### ISSUE TITLE
Dropdown Accessibility: Disabled items in a dropdown are not announced by a screen reader

### ISSUE THREAD

**Issue by jcaianirh** [patternfly/patternfly-react#5039]
**Describe the issue. What is the expected and unexpected behavior?**
When validating https://github.com/patternfly/patternfly-react/issues/4946 with a customer who uses JAWS, we noticed a bug in the dropdown component. When asked what items were in the Basic dropdown on the examples page, he repeated all the items present in the dropdown, but the disabled items were not repeated. When asked if he expected the disabled items in the dropdown h said that he expects the disabled items to be announced as disabled and not omitted from the list. 
**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

Must use JAWS to reproduce this issue.
**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug
